### PR TITLE
Remove library_bootstrap_structInfo.js. NFC.

### PR DESCRIPTION
**Pull request by sbc100**
The code generated by ./tools/gen_struct_info.py only seems to
depended on printf, there is no need for these other functions.

### CODE REVIEW

File: src/library_formatString.js [hunk 1]
```diff
@@ -4,6 +4,12 @@
  * SPDX-License-Identifier: MIT
  */
 
+
+#if BOOTSTRAPPING_STRUCT_INFO
+assert(!LibraryManager.library);
+LibraryManager.library = {}
```
Comment (kripken) [inline src/library_formatString.js:10]: `;` at end of line

Comment (kripken) [reply src/library_formatString.js:10]: Also please copy over the 3-line comment from the deleted code, as without that this looks a bit mysterious.

**Comment by sbc100** [comment]
Do you know why these dependencies used to be required by are no longer?

File: src/library_formatString.js (hunk 1 above)
Comment (sbc100) [reply src/library_formatString.js:12]: Done.

**Comment by kripken** [comment]
I'm not sure offhand. But the JS/C library impl was very different years ago when those were needed.

For malloc for example I guess we just use it from compiled code normally now.

**Pull request merged**